### DISCUSSION TITLE
Copyright on Images?

### DISCUSSION THREAD

**Discussion by ip6li** [Alovoa/alovoa#296] [General]
License section says: "All images are proprietary"

So following questions arises: 

* Who is the copyright holder on that images?
* What are the licensing conditions?

**Comment by Nonononoki** [comment]
It it's a logo, then it's copyrighted by the company of the logo. (e.g. the Facebook logo, Google logo...).
Other images that I do not own the copyright to have a license file where image is located. I am the copyright holder of the remaining images (currently only the Alovoa logo).
